src/terrain/util/transformers.clj

```diff
@@ -22,7 +22,7 @@
   (cond
     (string? v)     (string/blank? v)
     (sequential? v) (empty? v)
-    :else           true))
+    :else           (nil? v)))
```

Comment:
This change was necessary since `compojure-api` will coerce params into types other than strings and lists.